packages/client/wallets/aa/src/blockchain/BlockchainNetworks.ts

```diff
@@ -32,197 +40,151 @@ Chains not supported yet due fireblocks or zerodev doesn't supported
-        [Blockchain.OPTIMISM, "OPTIMISM"],
-        [Blockchain.ARBITRUM, "ARBITRUM"],
-        [Blockchain.GOERLI, "ETHEREUM"],
-        [Blockchain.SEPOLIA, "ETHEREUM"],
-        [Blockchain.MUMBAI, "MATIC"],
-        [Blockchain.ZKATANA, "ETHEREUM"],
+        ["ethereum", "ETHEREUM"],
+        ["polygon", "MATIC"],
+        ["bsc", "BNB_BSC"],
+        ["optimism", "OPTIMISM"],
+        ["arbitrum", "ARBITRUM"],
+        ["goerli", "ETHEREUM"],
+        ["ethereum-sepolia", "ETHEREUM"],
+        ["mumbai", "MATIC"],
+        ["zkatana", "ETHEREUM"],
     ]).get(chain)!;
 }
 
-export function getZeroDevProjectIdByBlockchain(chain: Blockchain) {
-    const zeroDevProjectId = new Map<Blockchain, string>([
-        [Blockchain.ETHEREUM, ZD_ETHEREUM_PROJECT_ID],
-        [Blockchain.POLYGON, ZD_POLYGON_PROJECT_ID],
-        [Blockchain.BSC, ZD_BSC_PROJECT_ID],
-        [Blockchain.OPTIMISM, ZD_OPTIMISM_PROJECT_ID],
-        [Blockchain.ARBITRUM, ZD_ARBITRUM_PROJECT_ID],
-        [Blockchain.GOERLI, ZD_GOERLI_PROJECT_ID],
-        [Blockchain.SEPOLIA, ZD_SEPOLIA_PROJECT_ID],
-        [Blockchain.MUMBAI, ZD_MUMBAI_PROJECT_ID],
-        [Blockchain.ZKATANA, ZD_ZKATANA_PROJECT_ID],
-    ]).get(chain);
+export function getZeroDevProjectIdByBlockchain(chain: BlockchainIncludingTestnet) {
+    const zeroDevProjectId = new Map([
+        ["ethereum", ZD_ETHEREUM_PROJECT_ID],
+        ["polygon", ZD_POLYGON_PROJECT_ID],
+        ["bsc", ZD_BSC_PROJECT_ID],
+        ["optimism", ZD_OPTIMISM_PROJECT_ID],
+        ["arbitrum", ZD_ARBITRUM_PROJECT_ID],
+        ["goerli", ZD_GOERLI_PROJECT_ID],
+        ["ethereum-sepolia", ZD_SEPOLIA_PROJECT_ID],
+        ["mumbai", ZD_MUMBAI_PROJECT_ID],
+        ["zkatana", ZD_ZKATANA_PROJECT_ID],
+    ]).get(chain)!;
+
     if (zeroDevProjectId == null) {
         throw new Error(`ZeroDev project id not found for chain ${chain}`);
     }
     return zeroDevProjectId;
 }
 
-export function getApiUrlByBlockchainType(chain: Blockchain): string {
+export function getApiUrlByBlockchainType(chain: BlockchainIncludingTestnet): string {
     const result = isTestnet(chain) ? CROSSMINT_STG_URL : CROSSMINT_PROD_URL;
     return result;
 }
 
-export function getWeb3AuthBlockchain(chain: Blockchain): TORUS_LEGACY_NETWORK_TYPE {
+export function getWeb3AuthBlockchain(chain: BlockchainIncludingTestnet): TORUS_LEGACY_NETWORK_TYPE {
     return isTestnet(chain) ? WEB3_AUTH_TESTNET : WEB3_AUTH_MAINNET;
 }
 
-export function isTestnet(chain: Blockchain): boolean {
-    const testnetKeys = new Set<keyof typeof BlockchainTestNet>(
-        Object.keys(BlockchainTestNet) as Array<keyof typeof BlockchainTestNet>
-    );
-
-    if (testnetKeys.has(chain.toUpperCase() as keyof typeof BlockchainTestNet)) {
-        return true;
-    } else {
-        return false;
-    }
+export function isTestnet(chain: BlockchainIncludingTestnet): boolean {
+    return (BLOCKCHAIN_TEST_NET as readonly string[]).includes(chain);
```

Comment:
to confirm–we tested this works?
I'm personally not certain how const arrays work with comparisons